validate/validate.proto

```diff
@@ -455,136 +455,148 @@ message BoolRules {
 // StringRules describe the constraints applied to `string` values
 message StringRules {
     // Const specifies that this field must be exactly the specified value
-    optional string const = 1;
+	optional string const = 1;
```

Comment:
Same here.

Reply:
Thanks a lot for reviewing, I deal with them latter ~